src/components/ExchangePage/index.tsx

```diff
@@ -827,16 +825,8 @@ function ExchangePage({ sendingInput = false, history, params }: ExchangePagePro
               <RowFixed>
                 <TYPE.black fontSize={14}>
                   {independentField === Field.INPUT
-                    ? slippageAdjustedAmounts[Field.OUTPUT]
-                      ? slippageAdjustedAmounts[Field.OUTPUT]?.toFixed(5) === '0.00000'
-                        ? '<0.00001'
-                        : slippageAdjustedAmounts[Field.OUTPUT]?.toFixed(5)
-                      : '-'
-                    : slippageAdjustedAmounts[Field.INPUT]
-                    ? slippageAdjustedAmounts[Field.INPUT]?.toFixed(5) === '0.00000'
-                      ? '<0.00001'
-                      : slippageAdjustedAmounts[Field.INPUT]?.toFixed(5)
-                    : '-'}
+                    ? slippageAdjustedAmounts[Field.OUTPUT]?.toSignificant(4) ?? '-'
```

Comment:
I ran into an issue with operator order before, does this need parentheses?

Reply:
i tried adding parens but prettier ate them. fwiw `??` should have higher precedence than the ternary, so it should be ok (although looking at it again, i guess it's weird that we're specifying the `'-'` fallback separately for each case, could just do it once...)